assets/templates/index.html

```diff
@@ -28,62 +28,64 @@
   <link rel="icon" type="image/x-icon" href="https://www.fromdoppler.com/favicon.ico" />
   <link rel="shortcut icon" type="image/x-icon" href="https://www.fromdoppler.com/favicon.ico" />
 </head>
+
 <body>
-<!-- rojo -->
-<div class="messages-container error">
-  <div class="wrapper">
-    <p class="ng-binding">Para que puedas enviar tu próxima Campaña, antes necesitamos verificar el origen de tus Suscriptores.</p>
-    <a href="#" class="button button--light button--tiny">Verificar Ahora</a>
-  </div>
-</div><!-- error -->
-<!-- amarillo -->
-<div class="messages-container warning">
-  <div class="wrapper">
-    <p>Thanks! We’ll contact you ASAP and if everything is fine, you’ll be able to send your Campaign.</p>
-  </div>
-</div><!-- warning-->
-<!-- verde -->
-<div class="messages-container message">
-  <div class="wrapper">
-    <p>Posees una cuenta gratis de 500  Suscriptores. Potencia tus Campañas con los beneficios de un Plan Pago.</p>
-    <a href="#" class="button button--light button--tiny">Comprar Ahora</a>
-  </div>
-</div><!-- message -->
-<header class="header-main">
-  <div class="header-wrapper">
+  <!-- rojo -->
+  <div class="messages-container error">
+    <div class="wrapper">
+      <p class="ng-binding">Para que puedas enviar tu próxima Campaña, antes necesitamos verificar el origen de tus Suscriptores.</p>
+      <a href="#" class="button button--light button--tiny">Verificar Ahora</a>
+    </div>
+  </div><!-- error -->
+  <!-- amarillo -->
+  <div class="messages-container warning">
+    <div class="wrapper">
+      <p>Thanks! We’ll contact you ASAP and if everything is fine, you’ll be able to send your Campaign.</p>
+    </div>
+  </div><!-- warning-->
+  <!-- verde -->
+  <div class="messages-container message">
+    <div class="wrapper">
+      <p>Posees una cuenta gratis de 500 Suscriptores. Potencia tus Campañas con los beneficios de un Plan Pago.</p>
+      <a href="#" class="button button--light button--tiny">Comprar Ahora</a>
+    </div>
+  </div><!-- message -->
+  <header class="header-main">
+    <div class="header-wrapper">
```

Comment:
@IgnacioRodrigues iIs it fine updating index.html indentation? Did @GusBaamonde and you initiate an indentation war?

Reply:
I am currently using atom-beautify for code identation. We do not talk about this, but I recommend to use this plugin because it is the same used in Sites. https://atom.io/packages/atom-beautify

Reply:
I use atom-beautify with identation size 2 space. @GusBaamonde  what space do you use?